### ISSUE TITLE
How to train the cosine metric learning model with the VeRi data set to be used in deep sort

### ISSUE THREAD

**Issue by jhonjam** [nwojke/cosine_metric_learning#79]
hi i am using yolov3 for vehicle detection and deep sort for tracking. however in some frames the detection of an object is lost, the deep sort algorithm generates a new id for the same object. I would like to do a new cosine metric model training to generate a .pb file to use in deep sort with the data set VeRI , however I have no idea what the format of the ground truth of objects is, in yolo the format is class, x1, y1, x2, y2, to train "cosine metric model" how would the gt_boxes of the images be? Could you explain or give me an idea about this. or someone who can share the trained model with me.

I have obtained information from other repositories, but none give detailed [information.](https://github.com/nwojke/cosine_metric_learning/issues/15#),

Thank you

**Comment by phungthengoc** [comment]
.

**Comment by Vuong02011996** [comment]
waitting

**Comment by seahawks8** [comment]
I was able to train my own model using custom dataset, and I am going to write a blog post soon for everyone.

**Comment by phungthengoc** [comment]
@seahawks8  i looking foward to you , i have some issue when trainning veri dataset

**Comment by lgold0525** [comment]
Hi @seahawks8, I am also very interested in training my own model using a custom dataset. Any updates on this?

**Comment by seahawks8** [comment]
I will do it eventually. Maybe next week.

**Comment by ngotra2710** [comment]
@seahawks8 Did you write a blog post about training deepsort model in Veri dataset? Please share it

**Comment by seahawks8** [comment]
not yet, too busy with work.

**Comment by kiraans** [comment]
@seahawks8 Please share the blog here, whenever you complete it!

**Comment by seahawks8** [comment]
sounds good

**Comment by akashAD98** [comment]
@seahawks8  have you written a blog. any updates

**Comment by sherlockchou86** [comment]
There is simple way to train the feature-extraction model using VeRi, just treat it as multi classes task.

Say using resnet50 in keras and add a Dense layer with softmax activation, one car is a category, every category has tons of images.

**Comment by IamExperimenting** [comment]
@seahawks8 , did you get a chance to write a blog?
